sw/host/ot_certs/src/x509.rs

```diff
@@ -75,6 +76,35 @@ fn asn1octets_to_vec(s: &Asn1OctetStringRef) -> Value<Vec<u8>> {
     Value::literal(s.as_slice().to_vec())
 }
 
+fn asn1time_to_string(time: &Asn1TimeRef) -> Result<Value<String>> {
+    // SAFETY: OpenSSL guarantees that an ASN1_TIME is in fact just a typedef for ASN1_STRING
```

Comment:
nit, as this does not have an unsafe block:
```suggestion
    // OpenSSL guarantees that an ASN1_TIME is in fact just a typedef for ASN1_STRING
```